build/ci/do_bats.sh

```diff
@@ -98,11 +105,21 @@ do_count_failures_tap() {
     fail_files_list="$fail_files_list $file"
     if [[ $1 -eq 1 ]]; then
         fail="$(echo "$tmp_out" | grep -c "^not ok")"
-    fi
+    elif [[ $1 -gt 123 ]]; then
+            # timed out tests
+         fail=1
+         echo "-> Timeout of ${TIMEOUT}s triggered while executing test.."
```

Comment:
There are no prints w/ echo (to control better the outout). use `logwarn` o `logdebug`.